modules/lambda/local.tf

```
@@ -0,0 +1,9 @@
+locals {
+  lambdas_path = "${path.module}/src/lambda_transcribe_conversation"
+  layers_path  = "${path.module}/../app/layers/nodejs"
+  layer_name   = "joi.zip"
+
+  statement_allow_writing_logs = ["arn:aws:logs:${var.aws_region}:${data.aws_caller_identity.current.id}:log-group:/aws/lambda/${var.name}:*"]
+  statement_allow_createing_log_groups = ["arn:aws:logs:${var.aws_region}:${data.aws_caller_identity.current.id}:*"]

```

Com uma variável intermediária, podemos deixar o código mais legível:

```suggestion
  logs_resource_prefix = "arn:aws:logs:${var.aws_region}:${data.aws_caller_identity.current.id}"
  logs_resource_writeable = ["${local.logs_resource_prefix}:log-group:/aws/lambda/${var.name}:*"]
  logs_resource_creatable = ["${local.logs_resource_prefix}:*"]
```